tests/test_wells4status_retrieval.py

```diff
@@ -91,17 +91,82 @@ def test_query(qcdb_test_session, mlwhdb_test_session, load_data4well_retrieval)
         _compare_dates(state.date_updated, expected_state[1])
 
 
-def test_inbox_retrieval(qcdb_test_session, mlwhdb_test_session):
+def test_inbox_wells_retrieval(
+    qcdb_test_session, mlwhdb_test_session, load_data4well_retrieval
+):
+    status = QcFlowStatusEnum.INBOX
 
-    factory = PacBioPagedWellsFactory(
+    paged_wells = PacBioPagedWellsFactory(
         qcdb_session=qcdb_test_session,
         mlwh_session=mlwhdb_test_session,
         page_size=10,
         page_number=1,
-        qc_flow_status=QcFlowStatusEnum.INBOX,
-    )
-    with pytest.raises(Exception, match=r"Not implemented"):
-        factory.create()
+        qc_flow_status=status,
+    ).create()
+    assert isinstance(paged_wells, PacBioPagedWells)
+    assert paged_wells.page_size == 10
+    assert paged_wells.page_number == 1
+    assert paged_wells.qc_flow_status == status
+    assert paged_wells.total_number_of_items == 8
```

Comment:
It's harder than I like to figure out how there are only 8 items in this inbox. Mutating fixtures and all that.

Reply:
Added comments, see https://github.com/wtsi-npg/npg_langqc/pull/93/commits/d0b604adcd9efcc1072c8b9339f09571a421909e